src/Parser/Mage/Frost/Modules/Features/Checklist.js

```diff
@@ -167,7 +167,7 @@ class Checklist extends CoreChecklist {
             when: combatant.hasTalent(SPELLS.THERMAL_VOID_TALENT.id),
           }),
           new Requirement({
-            name: "All Icicles into Glacial Spike",
+            name: "Glacial Spike Utilization",
```

Comment:
Maybe "Glacial Spike into Winter's Chill"? I've worried some about checklist items not being clear enough.